### PR TITLE
PMM-9562-PMM-server-IP-warning: add warning to kubernetesCluster

### PR DESCRIPTION
**Pull request by solovevayaroslavna**
https://github.com/percona-platform/grafana/pull/343
https://jira.percona.com/browse/PMM-9562

### CODE REVIEW

File: tests/DbaaS/verifyDBaaS_test.js [hunk 1]
```diff
@@ -254,7 +257,7 @@ Data(nameFields).Scenario('PMM-T456 Verify Create Cluster steps validation field
 
 Data(inputFields).Scenario('PMM-T456 Verify Create Cluster steps validation - field input validation @dbaas',
   async ({
-    I, dbaasPage, dbaasAPI, adminPage, current, dbaasManageVersionPage, dbaasActionsPage
```
Comment (puneet0191) [inline tests/DbaaS/verifyDBaaS_test.js:257]: dbaasManagerVersionPage was not used?

**Review by puneet0191 — APPROVED** [review]

**Comment by puneet0191** [comment]
@solovevayaroslavna could you please run dbaas-e2e pipeline with your custom branch and custom FB

File: tests/DbaaS/verifyDBaaS_test.js (hunk 1 above)
Comment (solovevayaroslavna) [reply tests/DbaaS/verifyDBaaS_test.js:257]: thanks for the comment, no, it's probably a mistake... I think my branch is outdated and I had to update it, I don't remember making such a change on purpose. I'll fix it

Comment (solovevayaroslavna) [reply tests/DbaaS/verifyDBaaS_test.js:257]: I checked, this parameter was really not used anywhere. so my linter removed it automatically

**Review by beatahandzelova — APPROVED** [review]

**Pull request closed without merging**